src/Angor/Client/Pages/Invest.razor

```diff
@@ -373,9 +374,7 @@
             {
                 var words = await passwordComponent.GetWalletAsync();
 
-                var accountInfo = storage.GetAccountInfo(_networkConfiguration.GetNetwork().Name);
-
-                var nostrPrivateKey = _derivationOperations.DeriveProjectNostrPrivateKey(words, accountInfo.InvestmentsCount + 1);
+                var nostrPrivateKey = _derivationOperations.DeriveProjectNostrInvestorPrivateKey(words, project.ProjectInfo.ProjectIdentifier);
```

Comment:
We have a pattern now to ask for passwords so this will fail
check how I did it in the create
https://github.com/block-core/angor/blob/main/src/Angor/Client/Pages/Create.razor#L381

Reply:
I made the call to password component on line 361